src/themes/material/Select.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Observer } from 'mobx-react'
+import _ from 'lodash/fp'
+import { Select as MaterialSelect, MenuItem } from '@material-ui/core'
+
+let Select = ({
+  options,
+  placeholder = 'Please Select...',
+  value = '',
+  ...props
+}) => (
+  <Observer>
```

Comment:
Is this needed, or can we use the HOC?

Reply:
we can use the HOC! https://github.com/mobxjs/mobx-react/blob/f9763b/test/stateless.test.js#L86